misk-hibernate/src/main/kotlin/misk/hibernate/DatabaseValidator.kt

```diff
@@ -0,0 +1,23 @@
+package misk.hibernate
+
+object DatabaseValidator : MiskSchemaParentValidator<MiskTable, MiskDatabase>() {
+
+  override fun process(errors: MiskSchemaErrors, dbSchema: MiskDatabase, ormSchema: MiskDatabase) {
```

Comment:
the universe is telling us these classes should be called DatabaseSchema or Schema

Reply:
DatabaseSchema
TableSchema
ColumnSchema 
?